cli/dev/DevServer.ts

```diff
@@ -60,7 +60,11 @@ export class DevServer {
   }
 
   async start() {
-    const { server } = await createHttpServer(this.port)
+    const circuitName = path.basename(
+      this.componentFilePath,
+      path.extname(this.componentFilePath),
+    )
```

Comment:
This is not the circuit name, it will always be `index` as the default file is `index.tsx`. Get the circuit name from the `package.json`.